bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/EnergyInfo.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.daikin.internal.api;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.time.DayOfWeek;
+import java.time.LocalDateTime;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import sun.security.util.Debug;
+
+/**
+ * Holds information from the get_day_power_ex call.
+ *
+ * @author Wouter Denayer <wouter@denayer.com> - Added to support energy reading
+ *         reading
+ *
+ */
+public class EnergyInfo {
+    public Optional<Double> energyHeatingToday = Optional.empty();
+    public Optional<Double> energyHeatingThisWeek = Optional.empty();
+    public Optional<Double> energyHeatingLastWeek = Optional.empty();
+    public Optional<Double> energyCoolingToday = Optional.empty();
+    public Optional<Double> energyCoolingThisWeek = Optional.empty();
+    public Optional<Double> energyCoolingLastWeek = Optional.empty();
+
+    private static final Logger logger = LoggerFactory.getLogger(EnergyInfo.class);
+
+    private EnergyInfo() {
+    }
+
+    public static EnergyInfo parse(String response) {
+
+        EnergyInfo info = new EnergyInfo();
+
+        logger.debug("parsing energy {}", response);
+
+        // /aircon/get_week_power_ex
+        // ret=OK,s_dayw=0,week_heat=1/1/1/1/1/5/2/1/1/1/1/2/1/1,week_cool=0/0/0/0/0/0/0/0/0/0/0/0/0/0
+        // week_heat=<today>/<today-1>/<today-2>/<today-3>/...
+        Map<String, String> responseMap = Arrays.asList(response.split(",")).stream().filter(kv -> kv.contains("="))
```

Comment:
You should use `InfoParser.parse(response);` The `InfoParser` class also has some new methods: `parseArrayofInt` that you could use to parse the specific values below. That might simplify the code too.

Reply:
I should remove this pull request. While I was in discussion/feedback on this PR another one was merged and made my work obsolete. Cf. https://community.openhab.org/t/daikin-binding-logging-energy-usage/96964

Reply:
I might be wrong, but if you're referring to #7708. It doesn't look like that pr overlapped with your pr. That pr added monthly reports, while you added daily/weekly data. That seems other data. So I think your changes are still relevant and from a quick look that pr added some methods that might could make your code simpler by using the added helper methods for parsing the data.